spicy/toolchain/bin/spicy-driver.cc

```diff
@@ -67,6 +49,26 @@ class SpicyDriver : public spicy::Driver, public spicy::rt::Driver {
     std::string opt_file = "/dev/stdin";
     std::string opt_parser;
 
+    void fatalError(const std::string& msg) {
+        hilti::logger().error(msg);
+        finishRuntime();
+        spicy::rt::done();
+        hilti::rt::done();
```

Comment:
... and here we call both `finishRuntime()` and the `done` functions. Unclear to me when either needs to be called now.

Reply:
`finishRuntime` is `hilti::Driver::finishRuntime` and in addition to finish hooks also invokes `hilti::rt::done`, so it should be preferable to plain `done` if available. We still need to explicitly shut down the spicy rt here.

I'll replace this with the following:

```.cpp
spicy::rt::done();
finishRuntime();
```